{{cookiecutter.repository_name}}/{{cookiecutter.package_name}}/model/model.py

```diff
@@ -24,14 +27,21 @@ def __init__(self):
             nn.Softmax(dim=1),
         )
 
-    def forward(self, prepared):
-        return model.PredictionBatch(logits=self.logits(prepared))
+    @property
+    def device(self):
+        return module_device(self)
 
-    def predictions(self, features: List[model.StandardizedImage]):
+    @validate_arguments
+    def forward(self, prepared: Tensor.dims("NCHW")):
+        return PredictionBatch(logits=self.logits(prepared))
+
+    def predictions(self, images: List[Image.Image]):
```

Comment:
maybe you just didnt get there yet but in case you forgot should be `lantern.Numpy` :)